lib/decidim/kids/engine.rb

```diff
@@ -39,10 +39,12 @@ class Engine < ::Rails::Engine
         Decidim::System::UpdateOrganizationForm.include(Decidim::Kids::System::OrganizationFormOverride)
         Decidim::System::UpdateOrganization.include(Decidim::Kids::System::UpdateOrganizationOverride)
         Decidim::System::RegisterOrganization.include(Decidim::Kids::System::RegisterOrganizationOverride)
+        Decidim::ParticipatorySpaceContext.include(Decidim::Kids::ParticipatorySpaceContextOverride)
       end
 
       initializer "decidim_kids.overrides", after: "decidim.action_controller" do
         config.to_prepare do
+          Decidim::Verifications::ApplicationController.include(Decidim::Kids::DisableMinorsParticipation)
           Decidim::ApplicationController.include(Decidim::Kids::WritesMinorLog)
           Decidim::Verifications::ApplicationController.include(Decidim::Kids::NeedsAdultPermission)
```

Comment:
This line should be deleted as the concern name has been changed and there is no concern named `NeedsAdultPermission` right now